src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/attributes/+page.svelte

```diff
@@ -64,10 +100,26 @@
                             </div>
                         </TableCell>
                         <TableCellText onlyDesktop title="Type">
-                            {`${attribute.type}${attribute.array ? '[]' : ''}`}
+                            {#if 'format' in attribute}
```

Comment:
Perhaps this needs to check if format is not empty because a datetime attribute can have an empty format:

<img width="946" alt="image" src="https://user-images.githubusercontent.com/1477010/230667871-16b2c811-c7c0-41d6-bceb-a82fe9f6a0c6.png">